apps/hubble-stats/components/Header/Header.tsx

```diff
@@ -38,16 +37,16 @@ const Header = () => {
           Icon={BlockIcon}
           iconSize="lg"
           iconClassName="stroke-blue-90 dark:stroke-blue-30"
-          value={tvl}
-          isLoading={tvl ? false : true}
+          value={tvlValue}
+          isLoading={tvlValue ? false : true}
         />
         <VolumeChip
           color="blue"
           Icon={BlockIcon}
           iconSize="lg"
           iconClassName="stroke-blue-90 dark:stroke-blue-30"
-          value={volume}
-          isLoading={volume ? false : true}
+          value={volumeValue}
+          isLoading={volumeValue ? false : true}
```

Comment:
`isLoading` should be applied only when we are loading the tvl and volume value, not when the tvl and volume are `undefined`. Maybe you can change the `value` prop here to "-" if those values are `undefined`.